packages/web3-eth-ens/src/lib/ResolverMethodHandler.js

```diff
@@ -41,14 +55,15 @@ function ResolverMethodHandler(registry) {
  * @param {function} callback
  * @returns {Object}
  */
-ResolverMethodHandler.prototype.method = function (ensName, methodName, methodArguments, callback) {
+ResolverMethodHandler.prototype.method = function (ensName, methodName, methodArguments, outputFormatter, callback) {
```

Comment:
The outputFormatter addition wouldn't be required.. you can just wrap the promise as on other places :-)

Reply:
(This ugly handler has to get refactored removed anyways.. just ping me.. I do have a solution somewhere on a handwritten paper)

Reply:
Not sure I follow, but yes this could be refactored at some point.